samples/Chat/src/app/utils/utils.ts

```diff
@@ -47,5 +45,4 @@ export const getBackgroundColor = (avatar: string): { backgroundColor: string }
   }
 };
 
-export const existsTopicName = (topicName?: string): boolean =>
-  !!topicName && topicName !== GUID_FOR_INITIAL_TOPIC_NAME;
+export const existsTopicName = (topicName?: string): boolean => !!topicName;
```

Comment:
this should also be removed as its not used at all anywhere (sorry to pile cleanup onto your PR)

Reply:
haha np! Updated :)